client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -162,15 +199,23 @@ class JetpackChecklist extends PureComponent< Props > {
 				>
 					<Task
 						{ ...JETPACK_CHECKLIST_TASK_PROTECT }
+						id="jetpack_task_protect"
+						onTaskTitleClick={ this.onTaskTitleClick }
+						selectedTaskId={ this.state.selectedTaskId }
```

Comment:
`Task` shouldn't have to know about `selectedTaskId`, see https://github.com/Automattic/wp-calypso/pull/33666/files#r292552161